java/test/jmri/jmrix/loconet/messageinterp/LocoNetMessageInterpretTest.java

```diff
@@ -6809,6 +6817,20 @@ public void testExpendedAccyWrites() {
         Assert.assertEquals("write 4: Ext Accy Addr 1 CV 12 to 0", ""
                 + "Extended Accessory Decoder CV Write: Address 1 CV 12 with 0.\n",
                 LocoNetMessageInterpret.interpretMessage(l, "LT", "LS", "LR"));
+
+        // [ED 0B 7F 51 15 3C 67 6D 03 5E 49] Extended Accessory Decoder CV 'Write': Address 500 CV 260, write 222.
+        l = new LocoNetMessage(new int[] {0xED, 0x0B, 0x7F, 0x51,
+            0x15, 0x3C, 0x67, 0x6D, 0x03, 0x5E, 0x49});
+        Assert.assertEquals("write 5: Ext Accy Addr 500 CV 260 to 222", ""
+                + "Extended Accessory Decoder CV Write: Address 500 CV 260 with 222.\n",
+                LocoNetMessageInterpret.interpretMessage(l, "LT", "LS", "LR"));
+        
+        // [ED 0B 7F 54 17 3C 6E 6D 03 5E 47] DT602 form Extended Accessory Decoder CV 'Write': Address 500 CV 260, write 222. 
+        l = new LocoNetMessage(new int[] {0xED, 0x0B, 0x7F, 0x54,
+            0x17, 0x3C, 0x6E, 0x6D, 0x03, 0x5E, 0x47});
+        Assert.assertEquals("write 6: Ext Accy Addr 500 CV 260 to 222", ""
+                + "Extended Accessory Decoder CV Write: Address 500 CV 260 with 222.\n",
+                LocoNetMessageInterpret.interpretMessage(l, "LT", "LS", "LR"));
     }
```

Comment:
Note that comment that says "DT602 Extended Accessory Decoder CV" access is _specifically_ a "DT602 Extended Accessory Decoder CV access to a Digitrax 7th-generation Accessory decoder".  I would suggest that you change the comment(s) to show something like that.